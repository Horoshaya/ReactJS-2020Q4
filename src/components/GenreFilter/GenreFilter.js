import React, { memo } from 'react';
import styles from './GenreFilter.css';
import GenreTab from '../GenreTab/GenreTab';

const InnerGenreSorting = ({ tabs }) => {
  return (
    <>
      {tabs.map((tab) => (
        <GenreTab key={tab.id} isActive={tab.isActive}>
          {tab.title}
        </GenreTab>
      ))}
    </>
  );
};

export const GenreFilter = memo(InnerGenreSorting);
