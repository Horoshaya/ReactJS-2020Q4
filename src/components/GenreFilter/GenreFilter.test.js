import React from 'react';
import rerender from 'react-test-renderer';
import GenreFilter from './GenreFilter';

describe('when GenreFilter', () => {
  test('then snapshot created', () => {
    const component = rerender.create(<GenreFilter tabs={[]} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
