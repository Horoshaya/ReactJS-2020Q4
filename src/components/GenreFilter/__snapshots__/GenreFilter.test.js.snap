// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when GenreFilter then snapshot created 1`] = `null`;
